truss/templates/triton/root/config.pbtxt.jinja

```diff
@@ -0,0 +1,34 @@
+name: "model"
```

Comment:
can we add a backlink to a properly formatted `config.pbtxt` this one seems minimal enough https://github.com/triton-inference-server/backend/blob/504abc9dc74f2f9aa907e561620c411ffbdb3f3c/examples/model_repos/minimal_models/batching/config.pbtxt#L4